integration/pod_userns_linux_test.go

```diff
@@ -304,6 +304,98 @@ func TestPodUserNS(t *testing.T) {
 	}
 }
 
+func TestIssue10598(t *testing.T) {
```

Comment:
Could you add a comment and the issue link to explain what is being tested here?

Reply:
Updated. Please take a look.